test-trigger.sh

```
@@ -0,0 +1 @@
+echo 'this is a test file and should not be in repo'
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)